docs/integrations/vercel-edge-config.md

```diff
@@ -32,14 +32,20 @@ If your Edge Config cannot store your project's configuration after the integrat
 
 ## Setup
 
+### DevCycle
+If you haven't already installed the DevCycle [Node.js](https://docs.devcycle.com/sdk/server-side-sdks/node/) or [Next.js](https://docs.devcycle.com/sdk/server-side-sdks/nestjs/) SDK you can follow the installation and usage guides for those SDKs in our [documentation here](https://docs.devcycle.com/sdk/). You can also find helpful setup information like where to find SDK keys in our [Quickstart Tutorial](https://docs.devcycle.com/introduction/quickstart).
+
+###Vercel
```

Comment:
move this to inside the "Setup SDK" section cuz by that point they'll have the integration set up and are about to start the code part, plus we'll have created an edge config for them that they can connect

Reply:
i think i moved the part you wanted moved. take another look

Reply:
it appears to be in the same place as before

Reply:
i was talking about moving the Vercel project connection part

Reply:
🤦